esmvaltool/cmorizers/data/cmor_config/ESACCI-LANDCOVER.yml

```
@@ -0,0 +1,56 @@
+attributes:
+  project_id: 'OBS'
+  dataset_id: 'ESACCI-LANDCOVER'
+  tier: 2
+  modeling_realm: sat
+  institution: ''
+  reference: 'esacci-landcover'
+  source: 'ftp://anon-ftp.ceda.ac.uk/neodc/esacci/land_cover/data/pft'
+  title: 'ESA CCI Land Cover'
+  version: 'v2.0.8'
+  comment: ''
+variables:
+  baresoilFrac:
+    mip: Lmon
+    long_name: 'BARE'
+    filename: ESACCI-LC-L4-PFT-Map-300m-P1Y-{year}-v2.0.8.nc
+  cropFrac:
+    mip: Lmon
+    long_name: 'GRASS-MAN'
+    filename: ESACCI-LC-L4-PFT-Map-300m-P1Y-{year}-v2.0.8.nc  
+  grassFrac:
+    mip: Lmon
+    long_name: 'GRASS-NAT'
+    filename: ESACCI-LC-L4-PFT-Map-300m-P1Y-{year}-v2.0.8.nc  
+  shrubs-bd:
+    mip: Lmon
+    long_name: 'SHRUBS-BD'
+    filename: ESACCI-LC-L4-PFT-Map-300m-P1Y-{year}-v2.0.8.nc
+  shrubs-be:
+    mip: Lmon
+    long_name: 'SHRUBS-BE'
+    filename: ESACCI-LC-L4-PFT-Map-300m-P1Y-{year}-v2.0.8.nc
+  shrubs-nd:
+    mip: Lmon
+    long_name: 'SHRUBS-ND'
+    filename: ESACCI-LC-L4-PFT-Map-300m-P1Y-{year}-v2.0.8.nc
+  shrubs-ne:
+    mip: Lmon
+    long_name: 'SHRUBS-NE'
+    filename: ESACCI-LC-L4-PFT-Map-300m-P1Y-{year}-v2.0.8.nc
+  trees-bd:
+    mip: Lmon
+    long_name: 'TREES-BD'
+    filename: ESACCI-LC-L4-PFT-Map-300m-P1Y-{year}-v2.0.8.nc
+  trees-be:
+    mip: Lmon
+    long_name: 'TREES-BE'
+    filename: ESACCI-LC-L4-PFT-Map-300m-P1Y-{year}-v2.0.8.nc
+  trees-nd:
+    mip: Lmon
+    long_name: 'TREES-ND'
+    filename: ESACCI-LC-L4-PFT-Map-300m-P1Y-{year}-v2.0.8.nc
+  trees-ne:
+    mip: Lmon
+    long_name: 'TREES-NE'
+    filename: ESACCI-LC-L4-PFT-Map-300m-P1Y-{year}-v2.0.8.nc                
```

We usually only list CMOR variables here, so in your case, that would be:

```suggestion
  shrubFrac:
    mip: Lmon
  treeFrac:
    mip: Lmon
```

You can then use special code in the formatter to make sure these variables are derived properly (very similar to what you did here: https://github.com/ESMValGroup/ESMValTool/blob/8f3c4c69f13288d987d0e90c334bff9466a2eba4/esmvaltool/cmorizers/data/formatters/datasets/esacci_soilmoisture.py#L144)